.github/CODEOWNERS

```diff
@@ -2,9 +2,9 @@
 /.changesets/ @apollographql/docs
 /apollo-federation/ @dariuszkuc @sachindshinde @goto-bus-stop @SimonSapin @lrlna @TylerBloom @duckki
 /apollo-federation/src/sources/connect @apollographql/connectors
-/apollo-router/ @apollographql/polaris @apollographql/atlas
+/apollo-router/ @apollographql/router-core @apollographql/fed-core
 /apollo-router/src/plugins/connectors @apollographql/connectors
-/apollo-router-benchmarks/ @apollographql/polaris @apollographql/atlas
-/apollo-router-scaffold/ @apollographql/polaris @apollographql/atlas
-/examples/ @apollographql/polaris @apollographql/atlas
-/.github/CODEOWNERS @apollographql/polaris @apollographql/atlas
+/apollo-router-benchmarks/ @apollographql/router-core @apollographql/fed-core
+/apollo-router-scaffold/ @apollographql/router-core @apollographql/fed-core
```

Comment:
We can remove this as it no longer exists
```suggestion
```

Reply:
Hrm.  Ok, undoing that commit. ;)